test/integration.test.js

```diff
@@ -949,7 +967,10 @@ test("Verify and notify failure", async (t) => {
     .get(
       `https://api.github.local/repos/${owner}/${repo}`,
       {
-        permissions: { push: true },
+        permissions: {
+          push: true,
+          maintain: true,
```

Comment:
@jedwards1211

Is `maintain` always `true`?? 🤔

Reply:
No but users for whom it's false may not be able to write issues, releases and pull requests, so I added a check that it's true in this PR.

Reply:
The props in permissions seem to be based upon this: https://github.nih.gov/about/features/access-permissions#:~:text=Maintain%3A%20Recommended%20for%20project%20managers,pull%20requests%20without%20write%20access

Reply:
Sounds great, I'll give this a whirl 🛞😁

So, how about a coverage for case when `maintain` is `false`, let's do that!? 😉

Reply:
There's a case for that in [verify.test.js](https://github.com/semantic-release/github/pull/897/files#diff-8baee4e69982184abf900a755dc4ca1bffe8acfc174b826186bc184d07b3187cR1447) now, do you want one in integration.test.js?

Reply:
Oh yes please 😁

Reply:
Okay is this sufficient? https://github.com/semantic-release/github/pull/897/commits/f5521226c5736d7b8c56a59d8249fd85a712b6cb#diff-6bb01a9664f0d2819bc4e2290b685dddc6dfb10fd786e7fa30933ae5ec57212bR74

Reply:
Let's gooooooo 💪